content/docs/other-guides/kustomize.md

```diff
@@ -0,0 +1,218 @@
+        ```
+        mkdir -p ${HOME}/bin
+        mv kustomize_*_${opsys}_amd64 ${HOME}/bin/kustomize
+        chmod u+x ${HOME}/bin/kustomize
+        ```
+
+1. Include the `kustomize` command in your path:
+
+      ```
+      export PATH=$PATH:${HOME}/bin/kustomize
+      ```
+
+## Modifying configuration before deployment
+
+kustomize lets you customize raw, template-free YAML files for multiple
+purposes, leaving the original YAML untouched and usable as is.
+
+You can use the following command to build and apply kustomize directories:
+
+```
+kustomize build <kustomization_directory> | kubectl apply -f -
+```
+
+The [Kubeflow manifests repo](https://github.com/kubeflow/manifests) contains
+kustomize build targets, each with a `base` directory. You can use kustomize to
+generate YAML output and pass it to kfctl. You can also make
+changes to the kustomize targets in the manifests repo as needed. 
+
+## Modifying the configuration of an existing deployment
+
+To customize the Kubeflow resources running within the cluster, you can modify
+the kustomize manifests in `${KFAPP}/kustomize`, where `KFAPP` is the directory 
+where you stored your Kubeflow configurations during deployment. Then run
+`kfctl apply`.
+
+For example, to modify settings for the Jupyter web app within your Kubeflow
+deployment:
+
+1. Edit the configuration file at `${KFAPP}/kustomize/jupyter-web-app.yaml`.
+
+1. Find and replace the parameter values:
+
+    ```
+    apiVersion: v1
+    data:
+    ROK_SECRET_NAME: secret-rok-{username}
+    UI: default
+    clusterDomain: cluster.local
+    policy: Always
+    prefix: jupyter
+    kind: ConfigMap
+    metadata:
+    labels:
+        app: jupyter-web-app
+        kustomize.component: jupyter-web-app
+    name: jupyter-web-app-parameters
+    namespace: kubeflow
+    ```
+
+1. Redeploy using kfctl:
+
+    ```
+    cd ${KFAPP}
+    kfctl apply k8s
+    ```
+
+    Alternatively, you can redeploy using kubectl directly:
+
+    ```
+    cd ${KFAPP}/kustomize
+    kubectl apply -f jupyter-web-app.yaml
+    ```
+
+### More examples
+
+For more usage examples, see the guide to [customizing Kubeflow on 
+GKE](https://www.kubeflow.org/docs/gke/customizing-gke/).
+
+## More about kustomize
+
+Below are some useful kustomize terms, from the 
+[kustomize glossary](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md):
+
+* **base:** A combination of a kustomization and resource(s). Bases can be
+  referred to by other kustomizations.
+
+* **kustomization:** Refers to a `kustomization.yaml` file, or more generally to
+  a directory containing the `kustomization.yaml` file and all the relative file 
+  paths that the YAML file references.
+
+* **overlay:** A combination of a kustomization that refers to a base, and a
+patch. An overlay may have multiple bases.
+
+* **patch:** General instructions to modify a resource.
+
+* **resource:** Any valid YAML file that defines an object with a kind and a
+metadata/name field.
```

Comment:
Indent this line to match **resource:**.

Reply:
Done.